Software/setup.sh

```diff
@@ -25,17 +17,20 @@ sudo apt-get --assume-yes install python3-setuptools
 
 sudo easy_install3 pip
 
+#install python3
 sudo apt-get --assume-yes install python3
```

Comment:
But python and python3 are different! unless installing python will automatically install python3, but I don't think so?